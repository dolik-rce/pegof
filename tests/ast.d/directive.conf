input ast.d/directive.peg
ast
