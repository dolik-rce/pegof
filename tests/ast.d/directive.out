GRAMMAR
  DIRECTIVE prefix (0 comments) "prefix"
  DIRECTIVE value (0 comments) "value"
  DIRECTIVE auxil (0 comments) "auxil"
  DIRECTIVE header (0 comments) {#define TEST 1}
  DIRECTIVE source (0 comments) {source}
  DIRECTIVE common (0 comments) {common}
  DIRECTIVE earlyheader (0 comments) {earlyheader}
  DIRECTIVE earlysource (0 comments) {earlysource}
  DIRECTIVE earlycommon (0 comments) {earlycommon\n    multiple\n    lines}
